tests/fixtures/cassettes/godaddy/IntegrationTests/test_Provider_authenticate.yaml

```diff
@@ -0,0 +1,32 @@
+interactions:
+- request:
+    body: !!python/unicode '{}'
+    headers:
+      Accept: [application/json]
+      Accept-Encoding: ['gzip, deflate']
+      Connection: [keep-alive]
+      Content-Length: ['2']
+      Content-Type: [application/json]
+      User-Agent: [python-requests/2.11.1]
+    method: GET
+    uri: https://api.ote-godaddy.com/v1/domains/000.biz
+  response:
+    body: {string: !!python/unicode '{"domainId":1002111,"domain":"000.biz","status":"TRANSFERRED_OUT","expires":"2016-06-14T23:59:59Z","expirationProtected":false,"holdRegistrar":false,"locked":true,"privacy":false,"renewAuto":true,"renewable":false,"transferProtected":false,"createdAt":"2015-06-15T13:10:43Z","authCode":"C36C3C6M26XE4DB5","subaccountId":"857256"}'}
```

Comment:
This "authCode" key in the body looks odd. Are you sure its not a variable you need to filter?